### DISCUSSION TITLE
Multiple tabs with same content?

### DISCUSSION THREAD

**Discussion by PGijsbers** [squidfunk/mkdocs-material#5702] [Q&A] [answered]
I want to use content tabs to show commands specific to each OS:
<img width="716" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/15890747/561a4f71-e824-47a1-9304-4e53b32ae6ce">
code:
```markdown
=== ":material-apple: MacOS"

    ```bash
    python runbenchmark.py randomforest 
    ```

=== ":simple-linux: Linux"

    ```bash
    python runbenchmark.py randomforest 
    ```

=== ":simple-windows: Windows"
    As noted above, we need to install the AutoML frameworks (and baselines) in
    a container. Add `-m docker` to the command as shown:
    ```bash
    python runbenchmark.py randomforest -m docker
    ```
    Future example usages will only show the UNIX invocation, but will still require
    Windows users to add `-m docker`.
```

However, more often than not MacOS and linux tabs share the same content. I was hoping I could do something like

```markdown
=== ":material-apple: MacOS"
=== ":simple-linux: Linux"

    my unix content

=== ":simple-windows: Windows"
    
    my windows content
```
so I don't need to duplicate my documentation. However, I can't find it in the [docs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#-or-even-me) and I tried multiple variations, but nothing works. Is there a way to achieve this?

**Comment by squidfunk** [comment, marked as answer]
There's currently no immediate way of deduplication. What we do is either merge tabs, or duplicate the content. Merging:

``` markdown
=== ":material-apple: MacOS, :simple-linux: Linux"

    my unix content

=== ":simple-windows: Windows"
    
    my windows content
```

**Reply by PGijsbers** [reply]
Thanks for letting me know. Unfortunately this does not play well with linked tabs; when selecting `linux` in one example where linux and macos are separate, it does not also select the `linux, macos` tab in a joint example. I think I will stick with duplication for now. Is it useful if I open an "idea" discussion?

**Reply by squidfunk** [reply]
This repository is not the right place to open an idea discussion, because everything that is Markdown processing is considered to be upstream. You might ask the maintainers of [Python Markdown Extensions](https://github.com/facelessuser/pymdown-extensions) on their input on this idea, who we work closely with.
